lib/licensee/projects/github_project.rb

```diff
@@ -39,8 +41,14 @@ def files
       end
 
       def load_file(file)
-        client.contents(@repo, path:   file[:path],
-                               accept: 'application/vnd.github.v3.raw').to_s
+        pathname = file[:path]
+        accept = 'application/vnd.github.v3.raw'
+        ref = @revision
+        if @revision
+          client.contents(@repo, path: pathname, accept: accept, ref: ref).to_s
+        else
+          client.contents(@repo, path: pathname, accept: accept).to_s
```

Comment:
can it be simplified with 

`client.contents(@repo, path: pathname, accept: accept, ref: ref).to_s`

and let the receiver deal with the fact that ref can be nil?

Reply:
I did that previously, but Octokit tries to stick `?ref` on the end of the github fetch URL if I do that, which causes problems in the tests (and possibly with the actual API response).

Reply:
gotcha